docs/types.md

```diff
@@ -783,6 +784,9 @@ All types of FileSpecs share some common fields:
 ### FileSpec -- Path
 
 A FileSpec with `type` equal to `path` points to an absolute or relative path accessible from the current machine.
+The path may include a glob pattern to target multiple files/directories.
+`dest` must be a directory if multiple files/directories are being copied (a new directory will be created if `dest` doesn't exist).
+An error occurs if multiple files/directories try to get copied when `dest` is an existing file.
```

Comment:
You can just drop this line.
Saying it "must" be a dir earlier implies an error on non-dir.